src/SourceBuild/content/repo-projects/Directory.Build.targets

```diff
@@ -345,16 +345,8 @@
     </ItemGroup>
   </Target>
 
-  <!--
-    Condition should be removed after source-build picks up version of Arcade that has
-    source-build infra changes for removal od inner-clone in VMR build. Until then some
-    repos will produce intermediate packages instead of repo manifest file.
-
-    https://github.com/dotnet/source-build/issues/3930
-  -->
   <Target Name="CopyRepoArtifacts"
           AfterTargets="Package"
-          Condition="Exists($(RepoManifestFile))"
           Inputs="$(RepoManifestFile)"
           Outputs="$(BaseIntermediateOutputPath)ArtifactsCopy.complete">
```

Comment:
Can you put an explicit Error condition here if the repo manifest file doesn't exist (rather than failing on the XmlPeek step)

Reply:
done